File: Java/src/e11/e_19.java
package e11;

public class e_19 {
    public static void main(String[] args) {
        // Cadena de caracteres
        String ch = "Hola Mundo";

        // Invertir la cadena
        String cadenaInvertida = invertirCadena(ch);

        // Mostrar la cadena invertida
        System.out.println("Cadena invertida: " + cadenaInvertida);
    }

    public static String invertirCadena(String str) {
        // Utilizar StringBuilder para invertir la cadena
        StringBuilder sb = new StringBuilder(str);
        sb.reverse();
        return sb.toString();
    }
}

// En este código, la función invertirCadena toma una cadena de entrada y
// utiliza la clase StringBuilder para invertirla. La salida esperada sería
// "odnuM aloH".

File: JS/11-20/e_19.js
// Escribe un programa en JS que permita invertir una cadena de caracteres.
// El programa debe invertir la variable 'ch' que contiene la frase 'Hola Mundo'.

// Cadena de caracteres
var ch = 'Hola Mundo';

// Invertir la cadena
var cadenaInvertida = ch.split('').reverse().join('');

// Mostrar la cadena invertida
console.log("Cadena invertida:", cadenaInvertida);

// En este código, split('') convierte la cadena en un array de caracteres, 
// reverse() invierte el orden de los elementos en el array y join('') une los caracteres invertidos en una cadena nuevamente. 
// La salida esperada sería "odnuM aloH".
